include/tvm/node/reflection.h

```diff
@@ -386,11 +386,13 @@ inline ReflectionVTable::Registry ReflectionVTable::Register() {
 
 inline void ReflectionVTable::VisitAttrs(Object* self, AttrVisitor* visitor) const {
   uint32_t tindex = self->type_index();
-  if (tindex >= fvisit_attrs_.size() || fvisit_attrs_[tindex] == nullptr) {
+  if (tindex >= fvisit_attrs_.size()) {
```

Comment:
This might cause some of the unintended behavior(since for certain object we might want VisitAttrs to exist and would like to detecting problem by seeing the error). Would be great if we can directly overload the getattr and dir on the python container object instead

Reply:
Thanks for your valuable advice! I re-submit a new commit to fix this bug by overloading the `__getattr__` and `__dir__`  function on the python container object. Since this commit only focuses on the front-end python container, I think it would not bring any unintended behaviors now.